game-app/game-core/src/main/java/games/strategy/triplea/ui/UnitChooser.java

```diff
@@ -25,11 +34,24 @@
 import javax.swing.JLabel;
 import javax.swing.JPanel;
 import javax.swing.JTextArea;
+import org.triplea.java.Postconditions;
 import org.triplea.java.collections.IntegerMap;
 
+/**
+ * This is the <code>JPanel</code> that shows the units to choose
+ * including the controls to make the choice.
```

Comment:
Please have a second look at this javadoc. It seems to be re-phrasing the name of the class a bit too much.

I would suggest to rephrase to be something like:
```
Displays a UI allowing the user to select how many of differing units. 

This can be used in a variety of ways, eg:
 * choosing which units take hits in a battle round.
 * choosing which units to place
```

Reply:
Javadoc will be improved in the next commit.